app/models/enterprise.rb

```diff
@@ -459,9 +459,8 @@ def validate_white_label_logo_link
 
   def image_url_for(image, name)
     return unless image.variable?
-    return image.variant(name).processed.url if image.attachment.service.name == :amazon_public
 
-    url_for(image.variant(name))
+    image_variant_url_for(image.variant(name))
```

Comment:
I'm wondering if would we need the same approach for `logo_url`, `image_url_for`, `image_url_for` (lines 299-309)?

I have not tested this PR, I'm asking [in the context of this one](https://github.com/openfoodfoundation/openfoodnetwork/pull/11106#issuecomment-1611877027).

Reply:
All good. `logo_url` calls `image_url_for` and that one handles this now. The error was on the `.service` call which we did only in these two methods I changed.

Reply:
Ahh gotcha ;-) Thanks for clarifying!